lib/src/components/NxFileUpload/stateful/NxStatefulFileUpload.tsx

```diff
@@ -10,7 +10,7 @@ import { StatefulProps as Props, statefulPropTypes } from '../types';
 
 export { Props };
 
-const NxStatefulFileUpload = forwardRef<HTMLDivElement, Props>(function NxStatefulFileUpload(props, ref) {
+const NxStatefulMultiFileUpload = forwardRef<HTMLDivElement, Props>(function NxStatefulMultiFileUpload(props, ref) {
```

Comment:
The changes in this file seem incorrect

Reply:
> The changes in this file seem incorrect

Thanks for catching that, definitely wrong - [5dce10c](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/5dce10c55958920ba51535184d5b7de3ea1bc53f)